ext/ClimaOceanSpeedyWeatherExt/bilinear_interpolator.jl

```diff
@@ -56,16 +64,16 @@ function coordinate_dataset(grid::AbstractGrid)
     lat,   lon   = two_dimensionalize(lat,   lon)
     lat_b, lon_b = two_dimensionalize(lat_b, lon_b)
 
-    lat_np = numpy[].array(lat)
-    lon_np = numpy[].array(lon)
+    return structured_coordinate_dataset(lat, lon, lat_b, lon_b)
+end
 
-    lat_b_np = numpy[].array(lat_b)
-    lon_b_np = numpy[].array(lon_b)
+function structured_coordinate_dataset(lat, lon, lat_b, lon_b)
 
-    return structure_coordinate_dataset(lat_np, lon_np, lat_b_np, lon_b_np)
-end
+    lat = numpy[].array(lat)
+    lon = numpy[].array(lon)
```

Comment:
what's this?

Reply:
We are adding weights from the Python xESMF package here to have an initial prototype working while ConservativeRegridding is under development.

Reply:
You should merge this first. Next you can develop a speedy-only simulation with prescribed SST. That will lead to the second goal.